backend/mock/bill.json

```diff
@@ -1,7 +1,7 @@
 {
-    "basePrintNo": "S7044",
+    "printNo": "S7044",
     "session": 2023,
-    "basePrintNoStr": "S7044-2023",
+    "printNoStr": "S7044-2023",
     "printNo": "S7044A",
```

Comment:
This file represents an API response. This field _does_ exist in the response, so let's not change it. Also, this change creates a duplicate key.

I'ma revert the file.